### PR TITLE
Add Media Queries section

### LINKED ISSUE #87: Media queries guidance

### PR DESCRIPTION
**Pull request by SebAshton**
This PR adds basic information about media queries and links to resources.

Closes #87

### CODE REVIEW

File: core-skills/frontend-web-development/resources/responsive-design.md [hunk 1]
```diff
@@ -1,2 +1,18 @@
-https://www.w3schools.com/html/html_responsive.asp
-https://developers.google.com/web/fundamentals/design-and-ux/responsive/
+# Media Queries
+
+An engineer should use a "mobile up" approach to translating a design into HMTL/CSS. The best way to achieve this is by using media queries.
```
Comment (craigjbass) [inline core-skills/frontend-web-development/resources/responsive-design.md]:
```suggestion
Using media queries will enable an HTML/CSS-based application (or website) to "respond" to changing screen widths.

In most situations where there is a requirement for the HTML/CSS-based user interface to work on a Mobile device, it makes sense to take a "mobile first"/"mobile up" approach. "Mobile first" refers to a discipline of HTML/CSS development which focusses on ensuring that the mobile experience is built before any larger sizes. These could be any number of, but not limited to, Tablet, Small Laptops, Larger Screens or Desktop size.
```

**Comment by craigjbass** [comment]
@kjdchapman are you owning this now - wondering if this can be merged or closed?

**Comment by kjdchapman** [comment]
If "this" is the frontend skill, not sure. Some reluctance to have single owners of these skills, some hesitation at me being sufficiently knowledgeable in this area.

If "this" is just this PR, then I think #100 does a better job of explaining this concept (_surprisingly, as I'm the author_), and would vote to close this.

**Comment by craigjbass** [comment]
This being the PR given you reference this pull request

**Comment by kjdchapman** [comment]
Closing this in favour of #100. Can always re-open if disputed.

**Comment by SebAshton** [comment]
Obviously I'm disputing @kjdchapman.

Not really though. Yours in much better. Great conversational writing style 👍

**Pull request closed without merging**